Frameworks/CoreText/CTFont.mm

```diff
@@ -902,4 +902,31 @@ CFDataRef CTFontCopyTable(CTFontRef font, CTFontTableTag table, CTFontTableOptio
 CFTypeID CTFontGetTypeID() {
     static CFTypeID __kCTFontTypeID = _CFRuntimeRegisterClass(&__CTFontClass);
     return __kCTFontTypeID;
+}
```

Comment:
Discussed offline - I pursued this route earlier, but exporting a struct through this many layers ended up being a much more ugly change. Leaving as is.

---
In reply to: [86256084](https://github.com/Microsoft/WinObjC/pull/1302#discussion_r86256084) [](ancestors = 86256084)